csharp/test/Ice/background/AllTests.cs

```diff
@@ -536,7 +536,7 @@ private static async Task closeConnection(Ice.ObjectPrx prx)
         for (int i = 0; i < 2; ++i)
         {
             configuration.readException(new Ice.SocketException());
-            BackgroundPrx prx = i == 0 ? background : (BackgroundPrx)background.ice_oneway();
+            BackgroundPrx prx = i == 0 ? background : background.ice_oneway();
```

Comment:
The proxy returned by ice_oneway() already had the correct type ("self"). The issue is the signature.